tests/sles4sap/publiccloud/hana_sr_test_secondary.pm

```diff
@@ -35,22 +36,29 @@ sub run {
     my $db_action = get_var('DB_ACTION', $run_args->{hana_test_definitions}{$self->{name}});
     croak('Database action unknown or not defined.') if ($db_action !~ /^(stop|kill|crash)$/);
 
-    if (($db_action eq 'crash') && get_var('HA_SBD_START_DELAY')) {
+    if (($db_action eq 'crash') && defined(get_var('HA_SBD_START_DELAY'))) {
```

Comment:
Is `defined` needed here? I can only see one other usage of this pattern in this repo

```
tests/jeos/firstrun.pm:
  
194:     if (is_generalhw && !defined(get_var('GENERAL_HW_VNC_IP'))) {
```

Reply:
Let me find some older runs... the test was considering this as false and would set the SBD delay to 0sec. 
Or let me run the test once again without it, what will happen.

Reply:
`defined` is important here depending on what the module should do when `get_var('HA_SBD_START_DELAY')` returns 0 or the empty string. Both 0 and the empty string will return true if evaluated with `defined` and false in other cases.

I see `setup_sbd_delay()` will skip empty string, but in the case of 0, it will try to set it.